src/textual/widget.py

```diff
@@ -1029,6 +1029,11 @@ def scrollbars_enabled(self) -> tuple[bool, bool]:
         enabled = self.show_vertical_scrollbar, self.show_horizontal_scrollbar
         return enabled
 
+    @property
+    def scrollbar_size(self) -> Size:
+        """The dimensions of the scrollbar stored in a Size object."""
```

Comment:
I'm finding this docstring a wee bit confusing; it makes a `Size` from `scrollbar_size_vertical` and `scrollbar_size_horizontal`, but they're two *different* scrollbars, as I understand it. Perhaps the docstring could do with being expanded to make it clear that it's about two different scrollbars and also make it clear which is in which `Size` attribute?

Reply:
It's poorly named, and could just be a tuple. I will clarify.